packages/react/src/components/CodeSnippet/CodeSnippet.js

```diff
@@ -253,6 +286,16 @@ CodeSnippet.propTypes = {
    */
   light: PropTypes.bool,
 
+  /**
+   * Specify the maximum number of rows to be shown when in closed view
+   */
+  maxClosedNumberOfRows: PropTypes.number,
```

Comment:
Would it make sense to use to "collapsed" instead of "closed'? I think there's precedent elsewhere in the component for the concept of it "expand"ing which would point to using "collapse"

```suggestion
  maxCollapsedNumberOfRows: PropTypes.number,
```

Reply:
The name can be changed if you want.
I went with closed, because comments in the scss used closed.

Reply:
changed